apps/common-app/src/examples/ScrollToExample.tsx

```diff
@@ -35,18 +77,59 @@ export default function ScrollToExample() {
         <Button onPress={scrollFromJS} title="Scroll from JS" />
```

Comment:
ScrollFromJS doesn't work with FlatList and FlashList (which is intended). I'd suggest removing this button when switching to these options

Reply:
I just forgot that lists on JS use `scrollToOffset` instead of `scrollTo`, which is used by `ScrollView`. I will refactor this component to work properly with lists as well. There is no need to remove the JS example for them.